extra/failover-testing/docker-compose.failover-server.yml

```diff
@@ -162,15 +146,15 @@ services:
             - CONFIG_PROP=config.properties
 
     mender-elasticsearch-2:
-        image: mendersoftware/elasticsearch:2.4
+        image: elasticsearch:5-alpine
```

Comment:
What about `elasticsearch`, is our fork only used in this test?

Reply:
as far as I can tell... and 2.4 is definitely out of date...

Reply:
and the https://hub.docker.com/r/mendersoftware/elasticsearch/tags looks interesting